examples/k3s/main.tf

```diff
@@ -0,0 +1,373 @@
+
+# VARIABLES # for you to edit!
+
+locals {
+  name          = "k3s"
+  pwd           = basename(path.cwd)
+  region        = "us-east-1"
+  ami           = "ami-0ff535566e7c13e8c"
+  instance_type = "m4.large"
+  vpc_cidr      = "10.0.0.0/16"
+  key_name      = "rajib_ssh_2"
```

Comment:
Do you want to generalize this too (as you did for autoscale) to not reveal your key name? Or maybe a better idea - have it come from the environment?